### PR TITLE
Making Content Great Again batch 1

### PR DESCRIPTION
**Pull request by hod-alpert**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

##related issue: 
https://github.com/demisto/etc/issues/23712

## Description
fixing content files so that they pass validation- batch 1

### CODE REVIEW

File: Packs/ANYRUN/TestPlaybooks/playbook-Detonate_File_-_ANYRUN_-_Test.yml [hunk 1]
```diff
@@ -83,24 +86,18 @@ tasks:
...
       username: {}
     separatecontext: false
-    view: |-
-      {
-        "position": {
-          "x": 50,
-          "y": 370
-        }
-      }
+    view: "{\n  \"position\": {\n    \"x\": 50,\n    \"y\": 370\n  }\n}"
```
Comment (Itay4) [inline Packs/ANYRUN/TestPlaybooks/playbook-Detonate_File_-_ANYRUN_-_Test.yml:95]: lets revert this change

File: Packs/ANYRUN/TestPlaybooks/playbook-Detonate_File_From_URL_-_ANYRUN_-_Test.yml [hunk 2]
```diff
@@ -73,29 +74,23 @@ tasks:
...
+        simple: https://www.python.org/downloads/release/python-373/
     separatecontext: false
-    view: |-
-      {
-        "position": {
-          "x": 50,
-          "y": 370
-        }
-      }
+    view: "{\n  \"position\": {\n    \"x\": 50,\n    \"y\": 370\n  }\n}"
```
Comment (Itay4) [inline Packs/ANYRUN/TestPlaybooks/playbook-Detonate_File_From_URL_-_ANYRUN_-_Test.yml:88]: lets revert this change

File: Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml [hunk 3]
```diff
@@ -61,6 +57,10 @@ configuration:
   name: proxy
   required: false
   type: 8
+- display: Incident type
```
Comment (Itay4) [inline Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:60]: why did we change this?

Comment (hod-alpert) [reply Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml:60]: It was a conflict resolving issue

Comment (Itay4) [reply Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml]: lets revert, it should be located where it was

Comment (hod-alpert) [reply Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml]: Done

**Comment by Itay4** [comment]
build fails on approved validation to be skipped
merging
